registry/handlers/api_test.go

```diff
@@ -57,6 +58,46 @@ func TestCheckAPI(t *testing.T) {
 	}
 }
 
+func TestURLPrefix(t *testing.T) {
```

Comment:
It's a copy of the TestCheckAPI code that adds a prefix and confirms it is being processed correctly. It feels odd to move this but leave TestCheckAPI here.